core/src/main/java/org/mapfish/print/http/MfClientHttpRequestFactoryImpl.java

```diff
@@ -166,14 +166,9 @@ protected Response executeInternal(@Nonnull final HttpHeaders headers) throws IO
                 }
             }
             if (this.request instanceof HttpEntityEnclosingRequest) {
-                HttpEntityEnclosingRequest entityEnclosingRequest = (HttpEntityEnclosingRequest) this.request;
-                Closer closer = Closer.create();
```

Comment:
What's was that Closer?

Reply:
Nothing was registered, so it was closing nothing.